src/pages/teams/[teamId]/dashboard/residents/[residentId]/index.tsx

```diff
@@ -103,116 +51,13 @@ const ResidentPage: NextPage<WithUser<ResidentPageProps>> = ({
-          paddingTop: '1.5em',
-        }}
-      >
-        <h2 className="lbh-heading-h3">
-          <SVGSymbol status="toReview" />
-          Pending review
-        </h2>
-
-        <EvidenceList>
-          {toReviewDocumentSubmissions &&
-          toReviewDocumentSubmissions.length > 0 ? (
-            toReviewDocumentSubmissions.map((ds) => (
-              <EvidenceTile
-                teamId={teamId}
-                residentId={residentId}
-                key={ds.id}
-                id={ds.id}
-                title={String(ds.documentType.title)}
-                createdAt={formatDate(ds.createdAt)}
-                fileSizeInBytes={ds.document ? ds.document.fileSizeInBytes : 0}
-                format={ds.document ? ds.document.extension : 'unknown'}
-                // purpose="Example form"
-                toReview
-              />
-            ))
-          ) : (
-            <h3>There are no documents to review</h3>
-          )}
-        </EvidenceList>
-      </div>
-
-      <div className="reviewed evidence-list">
-        <h2 className="lbh-heading-h3">
-          <SVGSymbol status="reviewed" />
-          Reviewed
-        </h2>
-
-        <EvidenceList twoColumns>
-          {reviewedDocumentSubmissions &&
-          reviewedDocumentSubmissions.length > 0 ? (
-            reviewedDocumentSubmissions.map((ds) => (
-              <EvidenceTile
-                teamId={teamId}
-                residentId={residentId}
-                key={ds.id}
-                id={ds.id}
-                title={String(ds.staffSelectedDocumentType?.title)}
-                createdAt={formatDate(ds.createdAt)}
-                fileSizeInBytes={ds.document ? ds.document.fileSizeInBytes : 0}
-                format={ds.document ? ds.document.extension : 'unknown'}
-                // purpose="Example form"
-              />
-            ))
-          ) : (
-            <h3>There are no reviewed documents</h3>
-          )}
-        </EvidenceList>
-      </div>
-
-      {rejectedDocumentSubmissions && rejectedDocumentSubmissions.length > 0 && (
-        <div className="rejected evidence-list">
-          <h2 className="lbh-heading-h3">
-            <SVGSymbol status="rejected" />
-            Rejected
-          </h2>
-
-          <EvidenceList twoColumns>
-            {rejectedDocumentSubmissions.map((ds) => (
-              <EvidenceTile
-                teamId={teamId}
-                residentId={residentId}
-                key={ds.id}
-                id={ds.id}
-                title={String(ds.documentType.title)}
-                createdAt={formatDate(ds.createdAt)}
-                fileSizeInBytes={ds.document ? ds.document.fileSizeInBytes : 0}
-                format={ds.document ? ds.document.extension : 'unknown'}
-                // purpose="Example form"
-              />
-            ))}
-          </EvidenceList>
-        </div>
-      )}
+      <ResidentDetailsTable resident={resident} />
+      <ResidentDocumentsTable
```

Comment:
We've extracted a lot of the logic that was on this page into its own components